src/App.tsx

```diff
@@ -1,14 +1,13 @@
 import GlobalStyles from './GlobalStyles'
+import { MainStyle } from './AppStyles'
```

Comment:
만약에, 스타일을 적용하시고 싶으실 때,

```
// AppStyles.ts
const MainStyle = styled.section`
  ${reset}
  width:360px;
  margin: 0 auto;
`

export default {
    MainStyle
}
```

로 적용해주시고, App.tsx 에서

```suggestion
import S from './AppStyles'
``

로 적용해주시고, 하단에 

```
  <S. MainStyle>
     // 중략
    </S.MainStyle>

```

이런식으로 작성해주시면, 만드신 컴포넌트와 스타일 컴포넌트가 혼동되지 않고 사용할 수 있을 것 같습니다.

Reply:
바로 아래에 댑변을 달아 주셨네요~ ㅋㅋㅋㅋ 감사해요~~